front/src/App.tsx

```diff
@@ -105,7 +99,12 @@ function App() {
             <Route
               index
               element={
-                <Protected component={<Dashboard />} redirectPath={prevLocation} minBeta={3} />
+                <Protected
+                  component={<Dashboard />}
+                  redirectPath={prevLocation}
+                  minBeta={3}
+                  requiredAbps={[["view_inventory", "view_shipments", "view_beneficiary_graph"]]}
```

Comment:
Shouldn't all three be required instead of just one? Now it means that a user with view_inventory permission can check beneficiary stats.

Reply:
It is ok. There is already a follow up ticket for this.

https://trello.com/c/uTYFpoQi